src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -406,7 +401,7 @@ private int getTotalPagesCount(User user, Map<String, String> searchCriteria, St
             .collect(Collectors.toList());
     }
 
-    private boolean isCaseOnHold(CaseDetails caseDetails) {
-        return caseDetails.getState().equals(CaseState.ONHOLD.getValue());
+    private boolean isCreatedState(CaseDetails caseDetails) {
+        return caseDetails.getState().equals(CREATED.getValue());
```

Comment:
Prefer `Objects.equals(A, B)` to avoid NPEs

Reply:
done